Ss/wandb_exporter (#21)
neptune-ai/scale-examples: utils/migration_tools/from_wandb/README.md

| Run metrics | run.scan_history() | run.<METRIC_NAME><sup>4</sup> |
| System metrics | run.history(stream="system") | run.runtime.<METRIC_NAME><sup>5</sup> |
| Console logs | Files > `output.log` | run.runtime.stdout<sup>6</sup> |
| Source code | Files > `code/` | run.source_code.files<sup>7</sup> |
| Requirements | Files > `requirements.txt` | run.source_code.requirements |
| Checkpoints | Files > `ckpt/`/`checkpoint/` | run.checkpoints |
| Other files | Files | run.files |
| All W&B attributes | run.* | run.wandb.* |

<sup>1</sup> Underscores `_` in a W&B project name are replaced by a hyphen `-` in Neptune  
<sup>2</sup> Passing the wandb.run.id as neptune.run.custom_run_id ensures that duplicate Neptune runs are not created for the same W&B run even if the script is run multiple times  
<sup>3</sup> Values are converted to a string in Neptune  
<sup>4</sup> `_step` and `_timestamp` associated with a metric are logged as `step` and `timestamp` respectively with a Neptune metric  
<sup>5</sup> `system.` prefix is removed when logging to Neptune. The `.` in the metric name is replaced with `/` to match the Neptune metric namespace format.  

> <sup>6</sup> Lines from the `output.log` file in W&B are logged as `stdout` in Neptune. Not all W&B runs have an `output.log` file, and even those that do don't necessarily have the same information as the console logs displayed in the _Logs_ section of the W&B run. Steps are logged as the line number of the `output.log` file, and the timestamp associated with each step is the current time of the script execution, not the timestamp logged to the W&B run.  

Reviewer: ```suggestion
<sup>6</sup> Lines from the `output.log` file in W&B are logged as `stdout` in Neptune. Not all W&B runs have an `output.log` file. Even when they do, the file doesn't necessarily reflect the information from  the console logs displayed in the _Logs_ section of the W&B run. Steps are logged as the line number of the `output.log` file, and the timestamp associated with each step is the current time of the script execution, not the timestamp logged to the W&B run.  
```
